verify_examples.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
+# Copyright 2020, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
```

Comment:
`set -euo pipefail` maybe could be a better safety net. `-u` is for bailing out on undefined variables (like in `rm -rf ${undefined}/*` ;) ). `-o pipefail` for bailing out in scenario like `failing_command | process_stdin`.